src/legacy/core_plugins/vis_default_editor/public/components/agg.tsx

```diff
@@ -303,6 +304,7 @@ function DefaultEditorAgg({
           onAggTypeChange={onAggTypeChange}
           setTouched={setTouched}
           setValidity={setValidity}
+          schemas={schemas}
```

Comment:
we need to pass schemas information down the component tree as its no longer available on the agg